util/task.py

```diff
@@ -82,15 +124,22 @@ class Task:
     def __init__(self):
         self.d = Discovergy(client_name)
         self.redis_client = redis.Redis(
-            host='localhost', port=6379, db=0)  # connect to server
+            host=redis_host, port=redis_port, db=redis_db)  # connect to server
 
     def login(self):
         """ Authenticate against the discovergy backend. """
 
         self.d.login(email, password)
 
     def populate_redis(self):
-        """ Populate the redis database with all discovergy data from the past. """
```

Comment:
Actually, from here on: I do not get which actual changes you are proposing. Please merge master into your branch and check which changes are actually remaining.